fs/base.py

```diff
@@ -1691,6 +1691,61 @@ def match(self, patterns, name):
         matcher = wildcard.get_matcher(patterns, case_sensitive)
         return matcher(name)
 
+    def match_glob(self, patterns, path, accept_prefix=False):
+        # type: (Optional[Iterable[Text]], Text, bool) -> bool
+        """Check if a path matches any of a list of glob patterns.
+
+        If a filesystem is case *insensitive* (such as Windows) then
+        this method will perform a case insensitive match (i.e. ``*.py``
+        will match the same names as ``*.PY``). Otherwise the match will
+        be case sensitive (``*.py`` and ``*.PY`` will match different
+        names).
+
+        Arguments:
+            patterns (list, optional): A list of patterns, e.g.
+                ``['*.py']``, or `None` to match everything.
```

Comment:
Looking at just the doc-strings, it's hard to see how `match_glob()` is different from `match()` ?

Reply:
The summaries (first sentences) differ: `Check if a name matches any of a list of wildcards.` and `Check if a path matches any of a list of glob patterns.`.